scripts/screener/screener.runner.ts

```diff
@@ -86,32 +78,9 @@ export async function screenerRunner(screenerConfig: ScreenerRunnerConfig) {
       : undefined,
   });
 
-  await notifyIntegration({
-    commit,
-    url: checkUrl.url,
-    status: 'in_progress',
-    project: screenerConfig.projectRepo,
-    branch: branchName,
-  });
-}
-
-export async function cancelScreenerRun(
```

Comment:
so this is not needed anymore as we will never run into such a scenario with this pr , correct ?

Reply:
yes, the `cancelScreenerRun` function is not needed anymore since the proxy is dealing with skipping or running screener tests now